tests/login.spec.js

```diff
@@ -1,14 +1,15 @@
-import { test, expect } from '@playwright/test';
-import CatalogPage from '../src/CatalogPage';
-import LoginPage from '../src/loginPage';
-
-
-test('Successfull login', async ({ page }) => {
-  await page.goto('https://www.saucedemo.com/');
-  let loginPage = new LoginPage(page);
-  await loginPage.submitLoginForm('standard_user','secret_sauce');
-  let catalogPage= new CatalogPage(page);
-  expect(catalogPage.productTitle).toHaveText('Products')
-})
-
-
+import { test, expect } from "@playwright/test";
+import CatalogPage from "../src/CatalogPage";
+import LoginPage from "../src/loginPage";
+
+test.beforeEach(async ({ page }) => {
+  console.log(`Running ${test.info().title}`);
+  await page.goto("https://www.saucedemo.com/");
+});
+
+test("Successfull login", async ({ page }) => {
+  const loginPage = new LoginPage(page); //mejor practica
```

Comment:
Remember to remove any comments that are no longer needed. In case this `//mejor practica` comment is still needed, do not mix languages, in this case, Spanish. Consistency across your code is important.